src/modules/admin/modules/users/services/userService.ts

```diff
@@ -0,0 +1,99 @@
+import FirestoreAdapter, { Doc } from '../../../../../database/firestoreAdapter'
+
+export interface User extends Doc {
+    name: String
+    email: String
+    photoURL: String
+    isApproved: boolean
+    isAdmin: boolean
+}
+
+export default class UserService {
+    public static readonly USERS: string = 'users'
+
+    constructor(private adapter: FirestoreAdapter) { }
+
+    // addUser(user: User): Observable<User> {
```

Comment:
these are just placeholders to just help me remember what functions I have to support. Will be removed in future PRs when I implement them